packages/react-router-dom/modules/__tests__/BrowserRouter-test.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import { BrowserRouter } from "react-router-dom";
 
 import renderStrict from "./utils/renderStrict";
 
+// resolved by jest
+import { BrowserRouter } from "react-router-dom";
```

Comment:
@mjackson A lot of the changes are me moving these imports and leaving a comment that it isn't a "real" import. Do you want this or should the imports stay at the top of the file?

Reply:
I decided to remove the comments and move the imports back to the top of the module.